docs/api.rst

```diff
@@ -469,7 +469,7 @@ to handle it, which defaults to print a traceback and ignoring the exception.
 
 .. function:: on_socket_event_type(event_type)
 
-    Called whenever a websocket event is received from the WebSocket.
+    Called whenever a WebSocket event is received from the WebSocket.
```

Comment:
Do we have to change this?

Reply:
Not necessarily but grammatically, yes.
<img width="415" alt="image" src="https://user-images.githubusercontent.com/89910983/181047772-11b4035a-b761-496d-a53a-decee4235a77.png">